file-formats/cfdf/type/zif_aff_cfdf_v1.intf.abap

```diff
@@ -0,0 +1,592 @@
+      "! <p class="shorttext">Business Scenario</p>
+      "! Business scenario
+      "! $required
+      "! $maxLength 30
+      business_scenario TYPE c LENGTH 30,
+
+      "! <p class="shorttext">Data Transfer</p>
+      "! Data transfer
+      "! $required
+      "! $maxLength 30
+      data_transfer     TYPE c LENGTH 30,
+    END OF ty_gs_extended_bus_scenario.
+  TYPES: ty_gt_extended_bus_scenario TYPE STANDARD TABLE OF ty_gs_extended_bus_scenario WITH KEY business_scenario.
+
+  TYPES:
+    "! <p class="shorttext">Extended Dynpro Context</p>
+    "! Extended dynpro context
+    BEGIN OF ty_gs_extended_dynpro_context,
+      "! <p class="shorttext">Dynpro Context</p>
+      "! Dynpro context
+      "! $required
+      "! $maxLength 30
+      dynpro_context   TYPE c LENGTH 30,
+
+      "! <p class="shorttext">Business Context</p>
+      "! Business context
+      "! $required
+      "! $maxLength 30
+      business_context TYPE zif_aff_types_v1=>ty_object_name_30,
+    END OF ty_gs_extended_dynpro_context.
+  TYPES ty_gt_extended_dynpro_context TYPE STANDARD TABLE OF ty_gs_extended_dynpro_context
+                                      WITH KEY dynpro_context business_context.
+
+  TYPES:
+    "! <p class="shorttext">Extended SOAP Service</p>
+    "! Extended SOAP service
+    BEGIN OF ty_gs_extended_soap_service,
+      "! <p class="shorttext">Service Interface</p>
+      "! Service interface
+      "! $required
+      "! $maxLength 30
+      service_interface    TYPE c LENGTH 30,
+
+      "! <p class="shorttext">Operation</p>
+      "! Operation
+      "! $required
+      "! $maxLength 30
+      operation            TYPE c LENGTH 30,
+
+      "! <p class="shorttext">Message Direction</p>
+      "! Message direction
+      "! $required
+      "! $maxLength 3
+      "! $values {@link zif_aff_cfdf_v1.data:co_soap_message_direction}
+      message_direction    TYPE ty_message_direction,
+
+      "! <p class="shorttext">Business Context</p>
+      "! Business context
+      "! $required
+      "! $maxLength 30
+      business_context     TYPE ty_business_context,
+
+      "! <p class="shorttext">Component Identifier</p>
+      "! Component identifier
+      "! $maxLength 5
+      component_identifier TYPE ty_component_identifier,
+
+      "! <p class="shorttext">External Namespace</p>
+      "! External namespace
+      "! $required
+      "! $maxLength 255
+      external_namespace   TYPE c LENGTH 255,
+
+      "! <p class="shorttext">External Field Name</p>
+      "! External field name
+      "! $required
+      "! $maxLength 120
+      external_field_name  TYPE c LENGTH 120,
+    END OF ty_gs_extended_soap_service.
+  TYPES: ty_gt_extended_soap_service TYPE STANDARD TABLE OF ty_gs_extended_soap_service
+                          WITH KEY business_context service_interface operation message_direction component_identifier.
+
+  TYPES:
+    "! <p class="shorttext">Code Description</p>
+    "! Code description
```

Comment:
```suggestion
    "! <p class="shorttext">Code Description Details</p>
    "! Code description details
```

Reply:
From my point of view details doesn't make sense, as only the descriptions itself together with the corresponding language and code value are contained, but no further details...